actionlist/jest/setup.js

```diff
@@ -0,0 +1,9 @@
+jest.mock('Linking', () => { // eslint-disable-line
```

Comment:
Why is this line being disabled? In general, you should only disable individual rules so other violations are still allowed to show up.